src/native/metrics/main.cpp

```diff
@@ -0,0 +1,55 @@
+#include <nan.h>
+
+#include "Collector.hpp"
+#include "EventLoop.hpp"
+#include "GarbageCollection.hpp"
+#include "Object.hpp"
+#include "Process.hpp"
+
+namespace datadog {
+  EventLoop eventLoop;
```

Comment:
It's a good idea to put these in an anonymous namespace, so they have internal linkage.
```
namespace datadog {
namespace {
  EventLoop eventLoop;
  GarbageCollection gc;
  Process process;
}
```